src/Entity/Activity.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ActivityRepository")
+ */
+class Activity
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $applicationDeadline;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $finalDeadline;
```

Comment:
What is the difference from `$applicationDeadline`?

Reply:
applicationDeadline - The deadline for job application
finalDeadline - The deadline for the job
We took it from Pentabee Project Info document